stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/avatar/Avatar.kt

```diff
@@ -64,60 +68,49 @@ public fun Avatar(
     initialsAvatarOffset: DpOffset = DpOffset(0.dp, 0.dp),
     onClick: (() -> Unit)? = null,
 ) {
-    if (imageUrl.isBlank()) {
-        InitialsAvatar(
-            modifier = modifier,
-            initials = initials,
-            shape = shape,
-            textStyle = textStyle,
-            onClick = onClick,
-            avatarOffset = initialsAvatarOffset,
+    var imageSize by remember { mutableStateOf(Size.ORIGINAL) }
+    Box(
+        modifier = modifier
+            .testTag("Stream_QuotedMessageAuthorAvatar") // This should not be here. Tied to the quoted component.
+            .onSizeChanged { size -> imageSize = Size(size.width, size.height) },
+    ) {
+        val cdnImageResizing = ChatTheme.streamCdnImageResizing
+        val data = remember { imageUrl.applyStreamCdnImageResizingIfEnabled(cdnImageResizing) }
```

Comment:
What about if the `imageUrl` needs to be sign by our customers implementation?
This URL won't be signed and no picture will be shown at all